server/server.js

```diff
@@ -0,0 +1,90 @@
+var express = require('express');
+var app = express();
+var port = 3000;
+var cors = require('cors')
+var task = require("./task");
+app.use(cors());
+// app.use(express.static(__dirname + '../html'));
+// app.use(express.static(__dirname + '../js'));
+// app.use(express.static(__dirname + '../node_modules'));
+
+
+//express
+var bodyParser = require("body-parser");
+app.use(bodyParser.json());
+
+app.post("/todo", function(req, res,next) {
+          task.create({"title": req.body.title,"dueDate": req.body.dueDate,"address": req.body.address,"lat": req.body.lat,"lng": req.body.lng}, function(err, task) {
```

Comment:
I don't think you need quotes around the keys, do you?

Maybe moving the hash to a different line would make this more readable too. What do you think?